src/olympia/devhub/forms.py

```diff
@@ -198,7 +198,8 @@ def save(self, *args, **kw):
         builtin = self.cleaned_data['builtin']
         if builtin == '':  # No license chosen, it must be an unlisted add-on.
             return
-        if builtin != License.OTHER:
+        is_other = builtin == License.OTHER
+        if not is_other:
```

Comment:
❤️ I'd call that `is_other_license` though I think 🤔

Reply:
I could but that will make the `if` on line 212 overflow and I know how much you dislike the double indentation needed on the following line 😜